docs/getting-started/installation-and-upgrade/install-upgrade-on-a-kubernetes-cluster/install-upgrade-on-a-kubernetes-cluster.md

```diff
@@ -242,6 +242,12 @@ In the following command,
 - Set `letsEncrypt.ingress.class` to whatever your ingress controller is, e.g., `traefik`, `nginx`, `haproxy`, etc.
 - For Kubernetes v1.25 or later, set `global.cattle.psp.enabled` to `false` when using Rancher v2.7.2-v2.7.4. This is not necessary for Rancher v2.7.5 and above, but you can still manually set the option if you choose.
 
+:::warning
+
+When the `agent-tls-mode` setting is set to `strict` (including on default, new installs of 2.9), you must supply the `--privateCA=true` flag and upload Let's Encrypt's Certificate Authority as outlined in [Adding TLS Secrets](../resources/add-tls-secrets.md). Information on identifying Let's Encrypt's Root CA can be found in the Let's Encrypt [docs](https://letsencrypt.org/certificates/). If this isn't done, then Rancher may fail to connect to new or existing downstream clusters.
```

Comment:
```suggestion
When the `agent-tls-mode` setting is set to `strict` (even by default, on new installs of 2.9), you must supply the `--privateCA=true` flag and upload the Let's Encrypt's Certificate Authority as outlined in [Adding TLS Secrets](../resources/add-tls-secrets.md). Information on identifying the Let's Encrypt's Root CA can be found in the Let's Encrypt [docs](https://letsencrypt.org/certificates/). If this isn't done, Rancher may fail to connect to new or existing downstream clusters.
```

Reply:
I re-worded the part which specifies that this is the default on new installs of 2.9. However, I left off the "the" suggestion - keep in mind that there's a `'s` in both cases of "Let's Encrypt". So I think the right choice would either be `the Let's Encrypt Certificate Authority` or `Let's Encrypt's Certificate Authority` (the CA belonging to Let's Encrypt). Since the second one is already there, I'm going to leave that as is. Let me know if you prefer the first (with the but without the `'`) and I can give it another look.